doc/port_auto_neg/port-auto-negotiation-design.md

```diff
@@ -396,9 +396,23 @@ To support validate interface speed on CLI side, a new field **supported_speeds*
 	; field                 = value
     ...
 	supported_speeds        = STRING                         ; supported speed list
+  speed                   = STRING                         ; operational speed
 
 An example value of supported_speeds could be "10000,25000,40000,100000".
 
+Before this feature, port speed in APP DB indicates both the configured speed and the operational speed. It is OK without this feature because port operational speed must be configured speed or port operational status is down. However, this is not true with this feature. Consider following flow:
+
+1. Configure port speed to 100G
+2. Configure advertised speed to 50G and enable auto negotiation
+3. Port operational speed turns to 50G
+4. Configure any port attribute, e.g mtu, portsyncd would put all port attributes to APP DB
```

Comment:
>portsyncd [](http://example.com/codeflow?start=42&length=9)

This statement is wrong. portmgrd is the right process.

Reply:
Hi Qi, portsyncd is also putting port configuration to APP DB. Please see https://github.com/Azure/sonic-swss/blob/45bdd1928e7d2e61d10fd1ce0da99abe1418bd13/portsyncd/portsyncd.cpp#L229

Reply:
@Junchao-Mellanox has agreed to raise PR to bring portmgrd instead of portsyncd for updating the port attributes in the AppDB